app/helpers/load_balancer_helper/textual_summary.rb

```diff
@@ -28,7 +28,8 @@ def textual_description
 
   def textual_listeners
     @record.load_balancer_listeners.map do |x|
-      "Load Balancer #{x.load_balancer_protocol}:#{x.load_balancer_port}, Instance #{x.instance_protocol}:#{x.instance_port}"
+      "Load Balancer #{x.load_balancer_protocol}:#{x.load_balancer_port_range_str}, Instance"\
```

Comment:
I actually don't know; in general the standard for port ranges seems to be <START>-<END> inclusive (or with a : instead of a -) but the prepending of the protocol may or may not be familiar.  Still, it seems readable enough to me so it seems worth including it in the helper